static/directives/manifest-vulnerability-view.html

```diff
@@ -187,15 +187,15 @@ <h3>Vulnerabilities</h3>
             <div class="severity-note" bo-if="vuln.scoreDivergence">
```

Comment:
Is this true if the distro score is "Unknown"?

Reply:
Yes, currently this is true.

Reply:
Sorry about my previous comment. But, after some testing,  I see that if the score is “Unknown”, it will not display any note. I believe that the severity-note is display if score divergence exists as stated [here](https://github.com/quay/quay/blob/master/static/directives/manifest-vulnerability-view.html#L187). And from [here](https://github.com/quay/quay/blob/master/static/directives/manifest-vulnerability-view.html#L187), I understand that if cvss score is null then score divergence is also null.